kvmagent/kvmagent/plugins/imagestore.py

```
@@ -136,6 +136,53 @@ def _get_progress(synced):
+        PFILE = linux.create_temp_file()
+        with linux.ShowLibvirtErrorOnException(vm):
+            cmdstr = '%s cbtbak -domain %s -volumes "%s" -bitmap "%s" -portrange "%s" > %s' % \
+                     (self.ZSTORE_CLI_PATH, vm.uuid, volumes, bitmapTimestamp, portRange, PFILE)
+            shell.call(cmdstr)
+            with open(PFILE) as fd:
+                linux.rm_file_force(PFILE)
+                json_data = fd.read()
+                return  _parse_json_and_update_mode(volume_infos, json_data)
+
+    def stop_vm_cbt_backup_jobs(self, vm, force=False):
+        with linux.ShowLibvirtErrorOnException(vm):
```

Ruff (0.8.2)

`linux` may be undefined, or defined from star imports

(F405)